lang/en.json

```
@@ -20,9 +20,9 @@
         "ManageGroups": "Manage Groups",
         "MenuGroupPermissions": "Group Permissions",
```

```suggestion
        "MenuGroupPermissions": "Permissions for Groups",
```
? (Avoids ~"Grouping of permissions")